azureml_deployment/README.md

```diff
@@ -0,0 +1,134 @@
+## Tutorial section 2: 
```

Comment:
The target audience is data scientists with no or limited experience with cloud tech or azure ml.
So keep this in mind in your explanations around the code. At the moment, you are to minimalistic for such a target audience.
Also, give them more background information. That way, it is more likely that the information will stick, for example:
- a schematic drawing of the deployment infra?
- what are the pieces of code doing?
- why are we using fast API over the default azure ml solution?
- where does my model fit in?
- etc.